index1.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="eng">
+  <head>
+    <meta charset="utf-8">
+    <title>Project: Rock, Paper, Scissors.</title>
+  </head>
+  <body>
+    <script>
+
+      function computerPlay() {
+        let randomDecimal = Math.random();
+        let randomNumToThree = randomDecimal * 3;
+        let roundedNum = Math.floor(randomNumToThree);
+
+        const weapons = ["rock", "paper", "scissors"];
```

Comment:
In JS, we use `'` for any string declarations, rather than `"`

Reply:
Amendments now made Kris. Thanks.